File: app/src/main/java/com/huanchengfly/tieba/api/models/CollectDataBean.java
package com.huanchengfly.tieba.api.models;

import com.huanchengfly.utils.GsonUtil;

public class CollectDataBean {
    public String pid;
    public String tid;
    public String status;
    public String type;

    public CollectDataBean(String pid, String tid, String status, String type) {
        this.pid = pid;
        this.tid = tid;
        this.status = status;
        this.type = type;
    }

    public CollectDataBean setPid(String pid) {
        this.pid = pid;
        return this;
    }

    public CollectDataBean setStatus(String status) {
        this.status = status;
        return this;
    }

    public CollectDataBean setTid(String tid) {
        this.tid = tid;
        return this;
    }

    public CollectDataBean setType(String type) {
        this.type = type;
        return this;
    }

    @Override
    public String toString() {
        return GsonUtil.getGson().toJson(this);
    }
}

File: app/src/main/java/com/huanchengfly/tieba/api/models/CollectDataBean.kt
package com.huanchengfly.tieba.api.models

import com.huanchengfly.utils.GsonUtil

class CollectDataBean(var pid: String, var tid: String, var status: String, var type: String) {
    fun setPid(pid: String): CollectDataBean {
        this.pid = pid
        return this
    }

    fun setStatus(status: String): CollectDataBean {
        this.status = status
        return this
    }

    fun setTid(tid: String): CollectDataBean {
        this.tid = tid
        return this
    }

    fun setType(type: String): CollectDataBean {
        this.type = type
        return this
    }

    override fun toString(): String {
        return GsonUtil.getGson().toJson(this)
    }

}